plugins/lookup/__init__.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Brian Scholer (@briantist)
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible.errors import AnsibleError
+# from ansible.plugins import AnsiblePlugin
+from ansible.plugins.lookup import LookupBase
+# from ansible import constants as C
+from ansible.utils.display import Display
+
+from ansible_collections.community.hashi_vault.plugins.plugin_utils.hashi_vault_plugin import HashiVaultPlugin
+
+display = Display()
+
+
+class HashiVaultLookupBase(HashiVaultPlugin, LookupBase):
+
+    def parse_kev_term(self, term, first_unqualified=None, plugin_name=None):
```

Comment:
Why not force `plugin_name` to always be specified?

Reply:
Hm, I thought I envisioned some reason why, can't remember now, but you're probably right. Will make that change and if I actually ran into something, could reconsider.